Swerve/SwerveTemplates/VisionBaseSwerve.java

```diff
@@ -45,11 +51,13 @@ public VisionBaseSwerve(SwerveModule[] swerveMods, Gyro gyro, SwerveDriveKinemat
         for (CameraConfig cameraConfig : cameraConfigs) {
             cameras.add(new Camera(cameraConfig));
         }
+
+        limelight = new Limelight(limelightConfig);
```

Comment:
should have option for no limelight on robot, not a guarantee that each vision swerve drive will have ll

Reply:
Yeah now if the limelightconfigs list is empty there will be no limelights in the code